content/pt-br/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -1,54 +1,113 @@
 ---
 reviewers:
-  - erictune
-  - soltysh
-  - janetkuo
+- erictune
+- soltysh
+- janetkuo
 title: CronJob
 content_type: concept
 weight: 80
 ---
 
-<!-- overview -->
+<!-- visão geral -->
 
-{{< feature-state for_k8s_version="v1.8" state="beta" >}}
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
-Um _Cron Job_ cria [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) em um cronograma baseado em tempo.
+Um _CronJob_ cria {{< glossary_tooltip term_id="job" text="Jobs" >}} em um cronograma recorrente.
 
-Um objeto CronJob é como um arquivo _crontab_ (tabela cron). Executa um job periodicamente em um determinado horário, escrito no formato [Cron](https://en.wikipedia.org/wiki/Cron).
+Um objeto CronJob é como uma linha em um arquivo _crontab_ (tabela cron). Executa uma tarefa periodicamente em um determinado cronograma, escrito no formato [Cron](https://en.wikipedia.org/wiki/Cron).
 
-{{< note >}}
-Todos os **CronJob** `schedule (horários):` são indicados em UTC.
-{{< /note >}}
+{{< caution >}}
 
-Ao criar o manifesto para um recurso CronJob, verifique se o nome que você fornece é um [nome de subdomínio DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
-O nome não deve ter mais que 52 caracteres. Isso ocorre porque o controlador do CronJob anexará automaticamente 11 caracteres ao nome da tarefa fornecido e há uma restrição de que o comprimento máximo de um nome da tarefa não pode ultrapassar 63 caracteres.
+Todos os horários da propriedade `schedule:` do  *CronJob* são baseadas no fuso horário do {{< glossary_tooltip term_id="kube-controller-manager" >}}.
 
-Para obter instruções sobre como criar e trabalhar com tarefas cron, e para obter um exemplo de arquivo de especificação para uma tarefa cron, consulte [Executando tarefas automatizadas com tarefas cron](/docs/tasks/job/automated-tasks-with-cron-jobs).
+Se a camada de gerenciamento do cluster executa o kube-controller-manager em Pods ou contêineres avulsos, o fuso horário configurado para o contêiner executando o kube-controller-manager determina o fuso horário que o controlador dos objetos CronJob utiliza.
 
+{{< /caution >}}
 
+Ao criar o manifesto para um objeto CronJob, verifique se o nome que você forneceu é um [nome de subdomínio DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
+O nome não pode ter mais que 52 caracteres. Esta limitação existe porque o controlador do CronJob adicionará automaticamente 11 caracteres ao final do nome de tarefa escolhido, e o comprimento máximo de um nome de tarefa não pode ultrapassar 63 caracteres.
```

Comment:
tarefa escolhida

Reply:
s/comprimento/tamanho (tamanho máximo de um nome)

Reply:
@rikatz could it be `nome escolhido para a tarefa` instead? I believe `escolhido` is referring to `nome` in this context.

Reply:
sure!

Reply:
Updated!